tests/Tests/500__jupyterhub/pvc-change_verification.robot

```diff
@@ -0,0 +1,76 @@
+*** Settings ***
+Documentation    PVC_CHANGE_VERIFICATION
+...                 Verify  if we can modify default PVC size
+...
+...
+...                 = Variables =
+...                 | Namespace | Required |    Jupyterhub configuration namespace|
+...                 | S_size    | Required |    Supported size |
+...                 | Ns_size   | Required  |   Not Supported PVC size |
+...                 | Size_code | Required |    Python code to check size in notebook |
+#
+Library         SeleniumLibrary
+Library         JupyterLibrary
+Resource         ../../Resources/ODS.robot
+Resource        ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Resource        ../../Resources/Page/ODH/JupyterHub/ODHJupyterhub.resource
+
+Test Teardown   PVC Size Test Teardown
+
+
+*** Variables ***
+${NAMESPACE}    redhat-ods-applications
+${S_SIZE}       15
+${SIZE_CODE}    import subprocess;
+...    int(subprocess.check_output(['df','-h', '/opt/app-root/src']).split()[8].decode('utf-8')[:-1])
+@{NS_SIZE}      0    -15    6.2    abc
+
+
+*** Test Cases ***
+Verify User Can Spawn Notebook With PVC Change
```

Comment:
the TC says to check if the user can Spawn the notebook, but none of the used keyword says `Spawn`

Reply:
**Verify Notebook Size** keyword has the spawn keyword inside it

Reply:
yes I saw it. But I mean maybe it would be better to use more self-explanatory keyword names

Reply:
I have chnaged the name of Tc check and let me know if it looks good